<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="use_sim_time" value="false"/>
    <node pkg="tf" type="static_transform_publisher" name="laser_broadcaster" args="0.019 0 0 3.14 0 0 base_link laser 100"/>
    <arg name="map_file" default="$(find object_search_project)/maps/search_map.yaml"/>
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
    <node pkg="tf" type="static_transform_publisher" name="localization_broadcaster" args="0 0 0 0 0 0 map base_link 100"/>
    <node pkg="object_search_project" type="search_manager" name="search_manager" output="screen"/>
    <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(find object_search_project)/rviz/object_search_project.rviz"/>

</launch>


